riscv-asm.md

```diff
@@ -214,7 +214,150 @@ ATTRIBUTE_VALUE := <string>
 
 #### `rvc`/`norvc`
 
-Enable/disable the C-extension for the following code region.
+This option will be deprecated soon after `.option arch` has been widely
+implemented on main stream open source toolchains.
+
+Enable/disable the C-extension for the following code region. This option is
+equivalent to `.option arch, +c`/`.option arch, -c`, but widely support by
+older toolchain versions.
+
+Alternative style:
+
+```
+.option push
+.option +c   # Alternative of .option rvc
```

Comment:
`.option arch, +c`

I think you do not intend to allow `.option +c`, right?

Reply:
Fixed